images/ironic-rhcos-downloader.yml

```
@@ -15,6 +15,8 @@ enabled_repos:
 - openstack-16-for-rhel-8-rpms
 for_payload: true
 from:
+  builder:
+  - member: openshift-base-rhel8
```

the Dockerfile is looking for a golang builder

```suggestion
  - stream: golang
```